general_motion_retargeting/motion_retarget.py

```diff
@@ -91,27 +90,27 @@ def __init__(
         self.rot_offsets1 = {}
         self.pos_offsets2 = {}
         self.rot_offsets2 = {}
-
         self.task_errors1 = {}
         self.task_errors2 = {}
 
         self.ik_limits = [mink.ConfigurationLimit(self.model)]
         if use_velocity_limit:
             VELOCITY_LIMITS = {k: 3*np.pi for k in self.robot_motor_names.keys()}
-            self.ik_limits.append(mink.VelocityLimit(self.model, VELOCITY_LIMITS)) 
+            self.ik_limits.append(mink.VelocityLimit(self.model, VELOCITY_LIMITS))
             
         self.setup_retarget_configuration()
-        
         self.ground_offset = 0.0
 
-    def setup_retarget_configuration(self):
-        self.configuration = mink.Configuration(self.model)
-    
+    def setup_retarget_configuration(self):    
         self.tasks1 = []
         self.tasks2 = []
         
         for frame_name, entry in self.ik_match_table1.items():
             body_name, pos_weight, rot_weight, pos_offset, rot_offset = entry
+            self.pos_offsets1[body_name] = np.array(pos_offset) - self.ground
+            self.rot_offsets1[body_name] = R.from_quat(
+                rot_offset, scalar_first=True
+            )
```

Comment:
Remnant. I put these dictionary assignments outside the if statement because otherwise its easy to get key/value errors when modifying the configs. This just creates offsets for all bodies even if they aren't used.